app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.includes(:actors).find(params[:id])
+    @actors = @movie.actors.order(age: :asc)
```

Comment:
Remember that we don't want to use `.order` or `.average` in the controller since they are AR methods beyond simple CRUD functions. These are MVC infractions as we want to be mindful of not putting too much complexity in controllers, especially when it comes to database operations.